src/app-locator.js

```diff
@@ -0,0 +1,98 @@
+import appIds from './known-app-ids'
```

Comment:
Thoughts on moving this file and `known-app-ids` to somewhere else, like `apps/`?

Reply:
It seems a bit odd to me, I see `apps/` as the directory for the “native apps”, while `known-app-ids` is about external apps.

Perhaps we could do this instead:

- Rename `apps/` into something a bit clearer, like `client-apps`.
- Recreate `apps/` and move everything related to actual apps in there.

Reply:
Hmm yes, this is true... maybe we can think on this for a bit. There are a number of icons and small configuration details that are also duplicated in regards to the known apps and maybe we can clean them all up at the same time.

Reply:
Also I think we should do this as a separate PR, as it’s more about the apps than the app locator feature.